src/Send/SendScreen/utils.ts

```diff
@@ -175,14 +185,19 @@ export const recomputeAll = async ({
           defaultAsset,
           selectedTokenInfo,
         )
+
         _fee = new MultiToken(yoroiUnsignedTx.unsignedTx.fee.values, {
           defaultNetworkId: yoroiUnsignedTx.unsignedTx.fee.defaults.networkId,
           defaultIdentifier: yoroiUnsignedTx.unsignedTx.fee.defaults.identifier,
         })
-        balanceAfter = tokenBalance.getDefault().minus(parsedAmount).minus(minAda).minus(_fee.getDefault())
+
+        balanceAfter = Quantities.diff(
+          defaultAssetAvailableAmount,
+          Quantities.sum([parsedAmount, minAda, _fee.getDefault().toString() as Quantity].map(Quantities.negated)),
```

Comment:
get fee from `yoroiUnsignedTx`

Reply:
can these calculations be converted to `YoroiAmounts.diff(balances, yoroiUnsignedTx.amounts)`?

Reply:
`balanceAfter` is `Quantity`. We need that value to calculate the `defaultAssetAmount` (Quantity). After that, both are displayed on the confirmation screen.

Do you think is worth change that value to `YoroiAmounts`?